app/controllers/rooms_controller.rb

```diff
@@ -0,0 +1,8 @@
+class RoomsController < ApplicationController
+  
+  def index
+    @rooms = Room.all
+    @hotel = Room.first.hotel_name
+    @count = Room.first.guest_count
+  end
+end
```

Comment:
Similar to my first comment that I had left above, you are only sending the hotel name and the guest count for the VERY FIRST room in your table. We want to display ALL of the rooms, and each room should display their respective hotel name and guest count. You can imagine, we might have multiple hotels with their own rooms. Right now, the way this is set up, you're only sending one hotels name and one rooms guest count.

So, having line 4 is all you need, since we really just want to send over ALL of our rooms to the view. Then, we'll iterate through that collection and get the respective hotels and guest counts for each room to display.